packages/dev/test/Flows.test.ts

```diff
@@ -228,7 +228,7 @@ options.forEach(params => {
           await approvalPaymaster.setRelayHub(rhub.address)
           await approvalPaymaster.setTrustedForwarder(await sr.getTrustedForwarder())
           await rhub.depositFor(approvalPaymaster.address, { value: (1e18).toString() })
-          relayClientConfig = { ...relayClientConfig, ...{ paymasterAddress: approvalPaymaster.address } }
+          relayClientConfig = { ...relayClientConfig, ...{ paymasterAddress: approvalPaymaster.address }, performDryRunViewRelayCall: false }
```

Comment:
why override defaults?
we should override defaults only if we're explicitly testing a rare case, and Flows is all about testing the "usual" cases.

Reply:
mistake. its only for a single test...